pfcwd/main.py

```diff
@@ -204,10 +204,12 @@ def interval(poll_interval):
             restoration_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('restoration_time'))
             if ((detection_time_entry_value != None) and (detection_time_entry_value < entry_min)):
                 entry_min = detection_time_entry_value
+                entry_min_str = "detection_time"
             if ((restoration_time_entry_value != None) and (restoration_time_entry_value < entry_min)):
                 entry_min = restoration_time_entry_value
-        if entry_min <= poll_interval:
-            print >> sys.stderr, "unable to use polling_interval = {}ms, value is bigger or equal to the minimum in PFCWD table".format(poll_interval)
+                entry_min_str = "restoration_time"
```

Comment:
same as above

Reply:
@neethajohn
please review and approve the changes.